README.md

```diff
@@ -82,7 +82,7 @@ Flags:
   -f, --config string               config file path
       --debug                       debug logs
   -h, --help                        help for agent-stack-k8s
-      --image string                The image to use for the Buildkite agent (default "ghcr.io/buildkite/agent-stack-k8s/agent:latest")
+      --image string                The image to use for the controller (default "ghcr.io/buildkite/agent-stack-k8s/agent:latest")
```

Comment:
I'll have to have a deeper look, but the default is the agent image not the controller image (specifically, where the agent image used to be uploaded, it's now `ghcr.io/buildkite/agent:latest`)